Lib9c/Action/ActionBase.cs

```diff
@@ -88,86 +98,94 @@ public AccountStateDelta(byte[] bytes)
             public IAccountStateDelta SetState(Address address, IValue state) =>
                 new AccountStateDelta(_states.SetItem(address, state), _balances);
 
-            public BigInteger GetBalance(Address address, Currency currency)
+            public FungibleAssetValue GetBalance(Address address, Currency currency)
             {
-                if (!_balances.TryGetValue((address, currency), out BigInteger balance))
+                if (!_balances.TryGetValue((address, currency), out (BigInteger, BigInteger) amount))
                 {
                     throw new BalanceDoesNotExistsException(address, currency);
                 }
 
-                return balance;
+                return new FungibleAssetValue(currency, amount.Item1, amount.Item2);
             }
 
-            public IAccountStateDelta MintAsset(Address recipient, Currency currency, BigInteger amount)
+            public IAccountStateDelta MintAsset(Address recipient, FungibleAssetValue value)
             {
                 // FIXME: 트랜잭션 서명자를 알아내 currency.AllowsToMint() 확인해서 CurrencyPermissionException
                 // 던지는 처리를 해야하는데 여기서 트랜잭션 서명자를 무슨 수로 가져올지 잘 모르겠음.
 
-                if (amount <= 0)
+                if (value <= new FungibleAssetValue(value.Currency))
                 {
-                    throw new ArgumentOutOfRangeException(nameof(amount));
+                    throw new ArgumentOutOfRangeException(nameof(value));
                 }
 
+                var nextAmount = GetBalance(recipient, value.Currency) + value;
+
                 return new AccountStateDelta(
                     _states,
-                    _balances.SetItem((recipient, currency),
-                        GetBalance(recipient, currency) + amount)
+                    _balances.SetItem(
+                        (recipient, value.Currency),
+                        (nextAmount.MajorUnit, nextAmount.MinorUnit)
```

Comment:
`FungibleAssetValue` is serializable.  Isn't it okay to use the `nextAmount` value in itself instead of a pair of `(nextAmount.MajorUnit, nextAmount.MinorUnit)`?

Reply:
Right, but it also relies on bencodex's serialization. so I need to write some translation codes like [it](https://github.com/planetarium/lib9c/pull/19/files#diff-01e3b7cdd708218e5c5c45ec04ab9196R256).😢

Reply:
Pair enough.  So would it be useful for this case if we change `FungibleAssetValue.RawValue`'s visibility from `internal` to `public`?

Reply:
I guess... yes maybe.

Reply:
Made a pull request for that: <https://github.com/planetarium/libplanet/pull/962>.

Reply:
[Libplanet 0.10.0-dev.20200821052050][1] has released!

[1]: https://www.nuget.org/packages/Libplanet/0.10.0-dev.20200821052050